src/test/java/saps/common/core/storage/swift/KeystoneV3IdentityPluginTest.java

```diff
@@ -18,17 +18,20 @@
   @Ignore
   public void createToken() throws Exception {
     Properties properties = new Properties();
-    FileInputStream input = new FileInputStream(CONFIG_FILE_PATH);
-    properties.load(input);
 
-    String url = properties.getProperty(SapsPropertiesConstants.Openstack.IdentityService.API_URL);
-    String projectId = properties.getProperty(SapsPropertiesConstants.Openstack.PROJECT_ID);
-    String userId = properties.getProperty(SapsPropertiesConstants.Openstack.USER_ID);
-    String password = properties.getProperty(SapsPropertiesConstants.Openstack.USER_PASSWORD);
+    try (FileInputStream input = new FileInputStream(CONFIG_FILE_PATH)) {
```

Comment:
are we missing a catch here?

Reply:
This is a try-with-resources block, is a syntax sugar to auto-close the resource (the FileInputStream in this case).

Reply:
does it depend on an specific jdk version onward?

Reply:
It is supported since jdk 7 (2011)

Reply:
yep, I'm old school, I guess. Nice!